lib/experimental/Widgets/Content/TasksList/TaskItem/index.tsx

```diff
@@ -56,16 +57,16 @@ export function TaskItem({
   return (
     <Wrapper onClick={handleOnClick} className={className}>
       {!hideIcon && (status === "due" || status === "no-due") && (
-        <DottedCircle
-          width={24}
+        <Icon
+          icon={DottedCircle}
           color={
             status === "no-due"
               ? "hsl(var(--neutral-40))"
               : "hsl(var(--neutral-50))"
           }
         />
       )}
-      {!hideIcon && status === "in-progress" && <InProgressTask />}
+      {!hideIcon && status === "in-progress" && <Icon icon={InProgressTask} />}
       <p className="mt-0.5 line-clamp-2 flex-1 font-medium">{task.text}</p>
       {!!task.badge && (
         <Badge
```

Comment:
I think you are ready to use your brand-new Tag component!

Reply:
Ohh true!